package-scripts.js

```diff
@@ -23,7 +23,7 @@ function test(testName, mochaParams) {
 module.exports = {
   scripts: {
     build: {
-      script: `browserify ./browser-entry --plugin ./scripts/dedefine --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' > mocha.js`,
+      script: `browserify -e browser-entry.js --plugin './scripts/dedefine' --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' -o mocha.js`,
```

Comment:
the AppVeyor failure is caused by this change; you may want to revert.

Reply:
It had worked fine for months, until I merged with master two days ago. And only fails when run on Windows.

Reply:
it’s the path-as-string I think that’s the windows problem. remove quotes I imagine